### PR TITLE
expression: support vectorized bitmap operations for `nulls` in `Column`

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
Introduce a new method `MergeNulls` for `Column`.

### What is changed and how it works?
```
BenchmarkMergeNullsVectorized-12         5000000               265 ns/op               0 B/op          0 allocs/op
BenchmarkMergeNullsNonVectorized-12       500000              3068 ns/op               0 B/op          0 allocs/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: util/chunk/column.go [hunk 1]
```diff
@@ -628,3 +628,14 @@ func (c *Column) CopyReconstruct(sel []int, dst *Column) *Column {
 	}
 	return dst
 }
+
+// OrNulls does the OR operation with all columns in the arguments.
+// The user should ensure that all these columns have the same length, and
+// data stored in these columns are fixed-length type.
+func (c *Column) OrNulls(cols ...*Column) {
```
Comment (SunRunAway) [inline util/chunk/column.go]: Add a unit test?

Comment (qw4990) [reply util/chunk/column.go]: My mistake and a unit test is added now.

File: util/chunk/column.go [hunk 2]
```diff
@@ -628,3 +628,17 @@ func (c *Column) CopyReconstruct(sel []int, dst *Column) *Column {
...
+
+// MergeNulls merges these columns' null bitmaps.
+// For a row, if any column of it is null, the result is null.
+// It works like: if col1.IsNull || col2.IsNull || col3.IsNull.
+// The user should ensure that all these columns have the same length, and
+// data stored in these columns are fixed-length type.
+func (c *Column) MergeNulls(cols ...*Column) {
+	for _, col := range cols {
+		for i := range c.nullBitmap {
+			// 1 is null while 0 is not null, so do AND operations here.
```
Comment (SunRunAway) [inline util/chunk/column.go]: `1 is not null while 0 is null`

Comment (zz-jason) [reply util/chunk/column.go]: @qw4990 `1` is not null. `0` is null. :joy:

Comment (qw4990) [reply util/chunk/column.go]: My mistake.
It is fixed now.

File: util/chunk/column_test.go [hunk 3]
```diff
@@ -885,3 +885,53 @@ func BenchmarkTimeVec(b *testing.B) {
...
+			rowResult.SetNull(i, lCol.IsNull(i) || rCol.IsNull(i))
+		}
+
+		for i := 0; i < 1024; i++ {
+			c.Assert(rowResult.IsNull(i), check.Equals, vecResult.IsNull(i))
+		}
+	}
+}
+
+func BenchmarkOrNullsVectorized(b *testing.B) {
```
Comment (SunRunAway) [inline util/chunk/column_test.go]:
```suggestion
func BenchmarkMergeNullsVectorized(b *testing.B) {
```

File: util/chunk/column_test.go [hunk 4]
```diff
@@ -885,3 +885,53 @@ func BenchmarkTimeVec(b *testing.B) {
...
+	cols := genNullCols(3)
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		cols[0].MergeNulls(cols[1:]...)
+	}
+}
+
+func BenchmarkOrNullsNonVectorized(b *testing.B) {
```
Comment (SunRunAway) [inline util/chunk/column_test.go]:
```suggestion
func BenchmarkMergeNullsNonVectorized(b *testing.B) {
```

**Comment by SunRunAway** [comment]
@qw4990 Please update PR description and benchmark result.

**Comment by qw4990** [comment]
> @qw4990 Please update PR description and benchmark result.

Updated.

**Pull request merged**